src/lib.rs

```diff
@@ -461,7 +461,7 @@ impl Vger {
         prim.quad_bounds = [c.x - radius, c.y - radius, c.x + radius, c.y + radius];
         prim.tex_bounds = prim.quad_bounds;
         prim.scissor = self.add_scissor() as u32;
-
+        prim.xform = self.add_xform() as u32;
```

Comment:
don't remove the newline, add your code under `prim.scissor`

Reply:
I have modified it